src/main/scala/freechips/rocketchip/amba/axi4stream/AXI4StreamModel.scala

```diff
@@ -103,248 +103,248 @@ case class AXI4StreamTransactionExpect
 )
 
 
-class AXI4StreamPeekPokeMaster(port: AXI4StreamBundle, tester: PeekPokeTester[_]) {
```

Comment:
Do we want to delete all the tests or just change MultiIOModule to Module?

Reply:
Chisel testers is getting deprecated. These must be removed.